app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[1] == command_list[2]:
+        return
+    else:
+        with open(command_list[1], "r") as file_in,\
+                open(command_list[2], "w") as file_out:
+            lines = file_in.readlines()
```

Comment:
You may also write the output of `readlines()` directly into `writelines()` without introducing a variable, if you want.

Reply:
Something like this?